howToTest.md

```diff
@@ -0,0 +1,76 @@
+[Jest](https://jestjs.io/docs/en/expect) main testing framework
```

Comment:
Can you add a linebreak after this line? 
Because in the preview both **Jest** and **Enzyme** are on the same line